.github/workflows/ci.yml

```diff
@@ -300,31 +299,6 @@ jobs:
           RUSTFLAGS: --cfg docsrs
           RUSTDOCFLAGS: --cfg docsrs -Dwarnings
 
-  loom:
```

Comment:
Perhaps it would be nice to have a task to build (not run) the tests with --cfg loom.
(It should interact well with the labeler action I mentioned in my comment.)

Reply:
Good point.

Reply:
:+1: we should definitely at least make sure that the loom tests still _compile_